src/Cli/dotnet/commands/dotnet-workload/update/WorkloadUpdateCommandParser.cs

```diff
@@ -30,6 +30,8 @@ internal static class WorkloadUpdateCommandParser
 
         public static readonly Option FromPreviousSdkOption = new Option<bool>("--from-previous-sdk", LocalizableStrings.FromPreviousSdkOptionDescription);
 
+        public static readonly Option AdManifestOnlyOption = new Option<bool>("--advertising-manifests-only", LocalizableStrings.AdManifestOnlyOptionDescription);
```

Comment:
@KathleenDollard did we settle on a syntax for the explicit gesture to update the advertising manifests? For now I've added this option to the update command, but it is easy to change.